### ISSUE TITLE
Search box case sensitive

### ISSUE THREAD

**Issue by anne123** [chembl/GLaDOS#1103]
There seem to be a number of issues with this:
The searches seem case sensitive in a number of cases:
e.g. lamotrigine vs LAMOTRIGINE
in this instance lowercase works but uppercase returns ALL the compounds.
(1) shouldn't any case work?
(2) if something isn't found it shouldn't return everything it should return nothing.
(3) the specific smiles example given below the search box doesn't work it also returns everything in the database.  The example given should work.  In contrast n1ccncc1 and NC1CCCCC1 do work.
(4) related to this can we have a better list of examples listed on the webpage.  I mentioned this in the gitbook David set up.

**Comment by nclopezo** [comment]
Hi Anne, 

For the examples under the search bar I created this issue: https://github.com/chembl/GLaDOS/issues/1095
Could you please write 4 examples and we set them up?
Would it be 'lapatinib, ERBB1' and which others?

**Comment by anne123** [comment]
Yes I'll have a think and get back to you - it would be good to find some examples that exemplify ChEMBL entities but in a clean way.  I'm thinking that Niki found dopamine confusing as it can be a compound and target

**Comment by mpaulam** [comment]
I found another example: MIRVETUXIMAB retrieves everything, mirvetuximab retrieves only 2 compounds

**Comment by Emma-Manners** [comment]
I'm not sure if you're still looking for examples but 'TRIMETAZIDINE' and 'TRIMEBUTINE' also retrieve everything. Trimetazidine and Trimebutine retrieve 3 and 2 records respectively.

**Comment by FionaEBI** [comment]
More examples: typing betamethasone (no quotes, lowercase) in the search box finds the correct 18 results:
![Screenshot 2019-10-16 at 10 51 39](https://user-images.githubusercontent.com/32895834/66908469-09b5fa80-f003-11e9-952d-ace7fd398d27.png)
And typing "BETAMETHASONE" (in quotes and uppercase) returns the correct results: 
![Screenshot 2019-10-16 at 10 52 54](https://user-images.githubusercontent.com/32895834/66908555-336f2180-f003-11e9-8e4a-bb681e0b3af0.png)
BUT typing BETAMETHASONE (no quotes) returns all the compounds in the database - which is incorrect since we know there should be 18 results:
![Screenshot 2019-10-16 at 10 54 07](https://user-images.githubusercontent.com/32895834/66908700-716c4580-f003-11e9-9a3d-d243d4bd64d1.png)
